File: ThreadsApp/ThreadsApp/Helpers/ConfigHelper.m
//
//  ConfigHelper.m
//  ThreadsApp
//
//  Created by Brooma Service on 02/04/2019.
//  Copyright © 2019 Brooma Service. All rights reserved.
//

#import "ConfigHelper.h"
#import "PlistUtils.h"

NSString* const kTHRConfigDictKey = @"THR_APP_CONFIG";
NSString* const kTHRServerUrlKey = @"THR_SERVER_URL";

@implementation ConfigHelper

+ (NSString *) getServerUrl {
    return [PlistUtils getStringFrom: kTHRConfigDictKey byKey:kTHRServerUrlKey];
}

@end

File: ThreadsApp/ThreadsAppSwift/Helpers/ConfigHelper.swift
//
//  ConfigHelper.swift
//  THRAppSwift-Polarbear
//
//  Created by Brooma Service on 14/05/2019.
//  Copyright © 2019 Brooma Service. All rights reserved.
//

import Foundation

@objc class ConfigHelper : NSObject {
    
    static let kTHRConfigDictKey = "THR_APP_CONFIG";
    static let kTHRServerUrlKey = "THR_SERVER_URL";
    
    @objc 
    class func getServerUrl() -> String? {
        return PlistUtils.getStringFrom(dictName: kTHRConfigDictKey, byKey: kTHRServerUrlKey)
    }
}
